idea-plugin/src/test/resources/log4j.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+    <appender name="console" class="org.apache.log4j.ConsoleAppender">
+        <param name="Target" value="System.out"/>
+        <layout class="org.apache.log4j.PatternLayout">
+            <param name="ConversionPattern" value="%-5p %c{1} - %m%n"/>
+        </layout>
```

Comment:
What does this append Log files to CI?

Reply:
it still logs to stdout. In my experience (with Travis) `stdout` is available for each job afterwards, but I'm not sure which CI provider you're using

Reply:
Yes perfect we're using Travis